configuration/src/main/java/io/camunda/configuration/beanoverrides/TasklistPropertiesOverride.java

```diff
@@ -48,9 +48,13 @@ public TasklistProperties tasklistProperties() {
     if (SecondaryStorageType.elasticsearch == database.getType()) {
       override.setDatabase("elasticsearch");
       override.getElasticsearch().setUrl(database.getElasticsearch().getUrl());
+      override.getElasticsearch().setUsername(database.getElasticsearch().getUsername());
+      override.getElasticsearch().setPassword(database.getElasticsearch().getPassword());
```

Comment:
:wrench: May be extract this to a method for better readability. - `populateElasticsearch` and `populateOpensearch`?